tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -451,6 +461,10 @@ def not_reached():
         assert body_runtime >= benchmark.benchmark_time(), \
             f"unexpectedly short runtime: {body_runtime} vs {benchmark.benchmark_time()}"
 
+        assert time() - time_before_swarm < swarm_runtime, (
```

Comment:
This check fails fast to avoid more confusing errors below if the swarm clients stopped too early (we expect them to be running at this point).